hector_gazebo_plugins/package.xml

```diff
@@ -20,7 +20,7 @@
 
   <!-- Dependencies needed to compile this package. -->
   <build_depend>roscpp</build_depend>
-  <build_depend>libgazebo7-dev</build_depend>
+  <build_depend>gazebo_ros</build_depend>
```

Comment:
Shouldn't this be the replaced by the new `gazebo_dev` package introduced in https://github.com/ros-simulation/gazebo_ros_pkgs/pull/571? Same as in https://github.com/ros-simulation/gazebo_ros_pkgs/blob/kinetic-devel/gazebo_plugins/package.xml.